shared/constants/swaps.ts

```diff
@@ -70,8 +70,8 @@ export const BNB_SWAPS_TOKEN_OBJECT: SwapsTokenObject = {
 } as const;
 
 export const MATIC_SWAPS_TOKEN_OBJECT: SwapsTokenObject = {
```

Comment:
Maybe we can re-name this variable too in a follow-up PR (no need to back port the rename though)